simtools/io_handler.py

```diff
@@ -96,3 +96,37 @@ def getInputDataFile(parentDir=None, fileName=None, test=False):
     else:
         filePrefix = Path(cfg.get("dataLocation")).joinpath(parentDir)
     return filePrefix.joinpath(fileName).absolute()
+
+
+def deleteOutputFile(fileName=None):
```

Comment:
I think you added these functions solely for the use of the test. However, as I explain below, they are not needed for the test. I also don't really see why they would be needed by a user of gammasim-tools. So, I suggest not adding them.